env/display.go

```diff
@@ -14,11 +15,29 @@ func Display(app string) error {
 	}
 	vars, err := c.VariablesList(app)
 	if err != nil {
-		return errgo.Mask(err, errgo.Any)
+		return errgo.Notef(err, "fail to list the environment variables")
 	}
 
 	for _, v := range vars {
 		fmt.Printf("%s=%s\n", v.Name, v.Value)
 	}
 	return nil
 }
+
+func Get(appName, variableName string) (string, error) {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return "", errgo.Notef(err, "fail to get Scalingo client to get an environment variable")
+	}
+	vars, err := c.VariablesListWithoutAlias(appName)
```

Comment:
I used `VariablesListWithoutAlias` rather than `VariablesList`. For instance, if a customer execute `env-get DATABASE_URL`, I want to display the actual value rather than `$SCALINGO_POSTGRESQL_VALUE`.